Doc/includes/sqlite3/execute_1.py

```diff
@@ -2,25 +2,24 @@
 
 con = sqlite3.connect(":memory:")
 cur = con.cursor()
-cur.execute("create table people (name_last, age)")
-
-who = "Yeltsin"
-age = 72
+cur.execute("create table lang (lang_name, lang_age)")
 
 # This is the qmark style:
-cur.execute("insert into people values (?, ?)", (who, age))
+name = "C"
```

Comment:
Can you get rid of name and age here as well?

Reply:
Sure!